flight/Modules/VtolPathFollower/vtol_follower_fsm.c

```diff
@@ -204,6 +208,7 @@ const static struct vtol_fsm_transition fsm_land_home[FSM_STATE_NUM_STATES] = {
 	[FSM_STATE_PRE_RTH_RISE] = {
 		.entry_fn = go_enable_rise_here,
 		.static_fn = do_ph_climb,
+		.timeout = 10 * MILLI,	/* Not sure this is good */
```

Comment:
Yah, and with the ability to be stuck at offsets probably some timeout is good.

We really need "not making [much] progress" alarms to do this well.  Maybe some of the alarm infrastructure i'm thinking about can be in the FSM (but there's probably other subsystems that want to use it, so maybe there's a clever way to factor it out).